demos/connector/neo4j/directed-graphsage-on-cora-neo4j-example.ipynb

```diff
@@ -0,0 +1,1158 @@
+       "    <tr>\n",
+       "      <th>4015</th>\n",
+       "      <td>Genetic_Algorithms</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1762</th>\n",
+       "      <td>Probabilistic_Methods</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2909</th>\n",
+       "      <td>Neural_Networks</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2943</th>\n",
+       "      <td>Rule_Learning</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>400</th>\n",
+       "      <td>Genetic_Algorithms</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>68</th>\n",
+       "      <td>Theory</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>270 rows × 1 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                    subject\n",
+       "id                         \n",
+       "893      Genetic_Algorithms\n",
+       "549         Neural_Networks\n",
+       "1212                 Theory\n",
+       "1137     Genetic_Algorithms\n",
+       "4015     Genetic_Algorithms\n",
+       "...                     ...\n",
+       "1762  Probabilistic_Methods\n",
+       "2909        Neural_Networks\n",
+       "2943          Rule_Learning\n",
+       "400      Genetic_Algorithms\n",
+       "68                   Theory\n",
+       "\n",
+       "[270 rows x 1 columns]"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "train_data[[\"subject\"]]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[[0. 1. 0. ... 0. 0. 0.]\n",
+      " [0. 0. 1. ... 0. 0. 0.]\n",
+      " [0. 0. 0. ... 0. 0. 1.]\n",
+      " ...\n",
+      " [0. 0. 0. ... 0. 1. 0.]\n",
+      " [0. 1. 0. ... 0. 0. 0.]\n",
+      " [0. 0. 0. ... 0. 0. 1.]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "target_encoding = feature_extraction.DictVectorizer(sparse=False)\n",
+    "\n",
+    "train_targets = target_encoding.fit_transform(train_data[[\"subject\"]].to_dict(\"records\"))\n",
+    "test_targets = target_encoding.transform(test_data[[\"subject\"]].to_dict(\"records\"))\n",
+    "\n",
+    "\n",
+    "print(train_targets)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now do the same for the node attributes we want to use to predict the subject. These are the feature vectors that the Keras model will use as input. The CORA dataset contains attributes 'w_x' that correspond to words found in that publication. If a word occurs more than once in a publication the relevant attribute will be set to one, otherwise it will be zero."
```

Comment:
This says "now we do the same", but there's nothing being done here. Maybe this paragraph doesn't need to be here in this notebook?